.github/workflows/deploy-sdk.yml

```diff
@@ -0,0 +1,213 @@
+name: Deploy SDK
+
+# Only run after a pull request has been merged. This is because
+# bot account runs write operations on the github repo to push a tag.
+on:
+  push:
+    branches: [main] # all branches where deployments currently occur. Make sure this list matches list of branches in  `.releaserc` file.
+
+permissions:
+  contents: write # access to push the git tag
+  issues: write # Bot creates an issue if there is an issue during deployment process
```

Comment:
```suggestion
  issues: write # Bot creates an issue if there is an issue during deployment process
  pull-requests: write # allow bot to make comments on PRs after they get deployed
```

Reply:
@levibostian we no longer use the `levibostian/action-sync-branches` which would create a PR (and required these permissions).  As far as I can see semantic-release also doesn't create a comment on the PR.

Do you still feel we need this?

Reply:
semantic-release [makes comments on PRs](https://github.com/customerio/customerio-ios/pull/230#issuecomment-1339556047). But, github now copies this feature:

![CleanShot 2023-02-13 at 12 07 33@2x](https://user-images.githubusercontent.com/2041082/218537769-8cfed194-684b-45c0-ac97-825da189d6d6.png)

....so, I suppose we don't need this feature anymore.

Reply:
Ah, I see. It's the `semantic-release/github` plugin. They also mention there's a `failComment` that may be worth keeping: https://github.com/semantic-release/github#failcomment

Shall I re-add?